events-api/protobuf/v1/event.proto

```diff
@@ -24,6 +24,9 @@ enum EventCategory {
   Pool = 1;
   Volume = 2;
   Nexus = 3;
+  Replica = 4;
+  Node = 5;
+  HA = 6;
```

Comment:
what is this HA?

Reply:
It is for high availability events. Currently a single event action "SwitchOver" is in the list of events.

Reply:
Changed it to "HighAvailability"